src/Config/FileSystemRuntimeConfigLoader.cs

```diff
@@ -164,9 +164,9 @@ private void OnNewFileContentsDetected(object? sender, EventArgs e)
     {
         try
         {
-            if (RuntimeConfig is not null && RuntimeConfig.IsDevelopmentMode())
+            if (RuntimeConfig is not null)
```

Comment:
by removing this condition, isnt it effectively enabling hot reload in production mode too?

Reply:
No, because we only ever file monitor if we start the engine in devmode.